src/Controller/CRUDController.php

```diff
@@ -330,6 +331,12 @@ public function editAction($id = null)
         $this->admin->setSubject($existingObject);
         $objectId = $this->admin->getNormalizedIdentifier($existingObject);
 
+        if (!$this->admin->getFormTabs()) {
```

Comment:
This is BC break cause at this point form can be **not** initialized yet, but moving call:
```
$form = $this->admin->getForm();
```
Before this `if` fixes that exception.

ps. mine admin controller don't overwrite any method of `CRUDController` just adds new like in docs.

Reply:
This was fixed in https://github.com/sonata-project/SonataAdminBundle/pull/5195

Reply:
Ah, just noticed fix: https://github.com/sonata-project/SonataAdminBundle/pull/5195